napari_allencell_annotator/_tests/fakes/fake_viewer.py

```diff
@@ -24,3 +24,23 @@ def alert(self, alert_msg: str) -> None:
 
     def get_layers(self) -> List[Layer]:
         return self._layers
+
+    def _get_all_points_layers(self) -> List[Points]:
+        return [layer for layer in self.get_layers() if isinstance(layer, Points)]
+
+    def create_points_layer(self, name: str, color: str, visible: bool, data: np.ndarray = None) -> Points:
+        points: Points = Points(data=None, name=name, color=color, visible=visible, ndim=6)
```

Comment:
`data=data`

Reply:
fixed in https://github.com/aics-int/napari-allencell-annotator/pull/90/commits/1797d93635cd7c64458a93d0022e3d4cc6f53157